tests/system/apmserver.py

```diff
@@ -167,7 +171,7 @@ def wait_until_ilm_setup(self):
 
     def wait_until_pipelines_registered(self):
         self.wait_until(lambda: self.log_contains("Registered Ingest Pipelines successfully"),
-                                                  name="pipelines registered")
+                        name="pipelines registered")
```

Comment:
we should sync the team on these settings.  I like the way this looks better but our automation doesn't seem to flag these and my editor wants to change them back.

Reply:
Oh, I was sure it got auto-changed when I ran `make fmt`.

Reply:
re-checked, running `make fmt` does change to what you see in the PR.